packages/mdc-list/_mixins.scss

```diff
@@ -409,6 +409,10 @@
     margin: 0;
     padding: 8px 0;
     list-style-type: none;
+
+    &:focus {
```

Comment:
curious why we didn't need this before?

Reply:
We didn't have `tabindex` on the list root before to be on focus.